docs/modules/ROOT/examples/env-vars.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: hazelcast.com/v1alpha1
+kind: Hazelcast
+metadata:
+  name: hazelcast
+spec:
+  clusterSize: 3
+  repository: 'docker.io/hazelcast/hazelcast-enterprise'
+  env:
+    - name: ENVIRONMENT
+      value: PROD
+    - name: AWS_ACCESS_KEY_ID
+      valueFrom:
+        secretKeyRef:
+          name: aws-secret
+          key: AWS_ACCESS_KEY_ID
+    - name: AWS_SECRET_KEY
+      valueFrom:
+        secretKeyRef:
+          name: aws-secret
+          key: AWS_SECRET_KEY
```

Comment:
```suggestion
```
I think we do not need to detailed YAML for the users, we support [Env for pod template](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.27/#envvar-v1-core) directly so we can mention this support.

Reply:
https://github.com/hazelcast/hazelcast-platform-operator-docs/pull/209/commits/7d6d98aa15f5807a2a4996da22f032068bb6d841